client/src/components/questions_answers/Answers.jsx

```diff
@@ -73,10 +88,10 @@ const Answers = ( { questionId } ) => {
             <div className="k-answer-date">{formatDate(answer.date)} | </div>
 
             <div className="k-answer-helpful">
-              Helpful? <span className="k-answer-yes-click" onClick={() => checkMarked(id)}>Yes</span> |  ({answer.helpfulness})
+              Helpful? <span className="k-answer-yes-click" style={{cursor: marked[id] ? "default" : "pointer"}} onClick={() => checkMarked(id)}>Yes</span> |  ({answer.helpfulness})
             </div>
             <div className="k-answer-report">
-              <span className="k-answer-report-click">Report</span>
+              <span className="k-answer-report-click" onClick={() => checkReprted(id)} style={{cursor: reported[id] ? "defualt" : "pointer"}} >{reported[id] ? "Reported" : "Report"}</span>
```

Comment:
typo: "checkReprted"

Reply:
Changed. Found another in the pointer on line 94. Also changed.